core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -1812,6 +1812,10 @@ class EMConfig {
     @Probability(activating = true)
     var probOfMutatingResponsesBasedOnActualResponse = 0.0
 
+    @Cfg("Number of threads for external request harvester")
+    @Experimental
```

Comment:
should add something like: "No more threads than numbers of processors will be used."

also, add constraints, ie, `@Min(1.0)`